gov_uk_dashboards/assets/dashboard.css

```diff
@@ -12637,4 +12638,48 @@ mark.expenditure {
     to an 8x8 pixel square so more of the pattern shows in the legend icon. For more information
     svg path command used above, take a look at
     https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorial/Paths#line_commands */
-}
\ No newline at end of file
+}
+
+.moj-side-navigation__list--nested {
+  padding-left: 20px;
+}
+
+.moj-side-navigation {
+  width: 20%;
+}
+
+.moj-side-navigation__item--expanded > a:after {
```

Comment:
I'm not sure if this works, but can we try grouping the shared styles like the below and just use the collapsed and expanded classes to control the arrows

```
.moj-side-navigation__item>a:after {
  border-bottom: 2px solid #1d70b8;
  border-right: 2px solid #1d70b8;
  content: "";
  height: 8px;
  position: relative;
  margin: 0 10px 0 2px;
  vertical-align: text-top;
  width: 8px;
  float: right;
}

.moj-side-navigation__item--expanded>a:after {
  transform: translateY(-35%) rotate(45deg) scale(1);
  transform: translateY(1px) rotate(225deg) scale(1)
}

.moj-side-navigation__item--collapsed>a:after {
  transform: translateY(-35%) rotate(225deg) scale(1);
  transform: translateY(1px) rotate(45deg) scale(1)
}
```